pandas/core/generic.py

```diff
@@ -1596,7 +1601,9 @@ def _get_label_or_level_values(self, key: str, axis: int = 0) -> np.ndarray:
 
         if self._is_label_reference(key, axis=axis):
             self._check_label_or_level_ambiguity(key, axis=axis)
-            values = self.xs(key, axis=other_axes[0])._values
+            values = self.xs(key, axis=other_axes[0])
+            if raw:
```

Comment:
why do u think u need raw at all?

Reply:
Because while most methods expect `_get_label_or_level_values` to return the underlying `arraylike`, we need the `Series` object itself to apply the keys functions to in `DataFrame::sort_values`. Otherwise we wouldn't be able to do things like `col.str.lower()`.

Reply:
no you dont - just try it, you can always wrap the return in a Series if you need
but this raw argument is pretty smelly and needs to go